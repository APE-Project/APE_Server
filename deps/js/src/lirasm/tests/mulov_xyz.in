; 1073741824 * 2 >= 2^31, and will overflow.
big = immi 1073741824
two = immi 2

res = mulxovi big two    ; overflow, so we exit here

; Ensure that 'big' and 'two' get their own registers and
; aren't shared with 'res'.  Also store 'res' so it isn't dead.
m = allocp 12
sti big m 0
sti two m 4
sti res m 8
x                       ; we don't exit here
