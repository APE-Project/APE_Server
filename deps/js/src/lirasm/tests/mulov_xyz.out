Exited block on line: 5
